app_spec/zomes/blog/code/src/post.rs

```diff
@@ -97,7 +97,7 @@ pub fn definition() -> ValidatingEntryType {
                 "%agent_id",
                 link_type: "recommended_posts",
                 validation_package: || {
-                    hdk::ValidationPackageDefinition::ChainFull
+                    hdk::ValidationPackageDefinition::Entry
```

Comment:
Just curious, did you change these because there was no reason to have them set to ChainFull, or does this actually have an impact?

Reply:
Basically because there is no reason to. I think it makes the tests easier to understand as well